src/utils/scores.test.ts

```diff
@@ -0,0 +1,101 @@
+const mockQuery = jest.fn(async () => {
+  return [
+    {
+      issue_id: '1',
+      repository: 'routing-repo',
+      product_area: 'Issues',
+      triaged_dt: { value: '2023-10-13T16:53:15.000Z' },
+      triage_by_dt: { value: '2023-10-12T21:52:14.223Z' },
+    },
+  ];
+});
+
+jest.mock('@google-cloud/bigquery', () => ({
```

Comment:
Just to be clear: this mock applies to all test cases, not just the ones in this file? In other words, is there anywhere in these tests that we're testing against a live BigQuery instance?

Reply:
None of the tests are against a live BigQuery instance